### PR TITLE
Doc: Extend the list of tools integrating kroki

### PR DESCRIPTION
**Pull request by cmorty**

### CODE REVIEW

File: docs/modules/setup/pages/third-party-tools.adoc [hunk 1]
```diff
@@ -37,3 +45,22 @@ Add any Kroki URL after `https://niolesk.top/#` and you'll be able to edit your
...
+
+{url-gitlab}[Gitlab] can {url-gitlab-int}[use Kroki to render Images] in Markdown, AsciiDoc, reST and Textile documents.
+
+== Sphinx Documentation
+
+There is an {url-sphinx-int}[Kroki integration] for the {url-sphinx}[Sphinx documentation generator].
+
+== Commercial products
+
+* {url-conf-kroki}[Kroki Diagrams for Confluence]
```
Comment (Mogztter) [inline docs/modules/setup/pages/third-party-tools.adoc:66]: I wasn't aware of this commercial offering and to be honest I don't want to include it.
They (try) to make money off the public instance (https://kroki.io) which is not fair.

I wouldn't mind if they were using their own instance but using the public instance (kindly hosted by Exoscale) in a commercial product is wrong!

Comment (cmorty) [reply docs/modules/setup/pages/third-party-tools.adoc:66]: No worries (I totally support you in this). I did a quick google search and found them, but didn't really look into it. Did you contact them?

File: docs/modules/setup/pages/third-party-tools.adoc [hunk 2]
```diff
@@ -3,6 +3,14 @@
...
 :url-keenwrite: https://github.com/DaveJarvis/keenwrite/
 :url-niolesk: https://niolesk.top/
+:url-vscode: https://code.visualstudio.com/
+:url-vscode-asciidoctor: https://marketplace.visualstudio.com/items?itemName=asciidoctor.asciidoctor-vscode
+:url-vscode-asciidoc-slides: https://marketplace.visualstudio.com/items?itemName=flobilosaurus.vscode-asciidoc-slides
+:url-gitlab: https://about.gitlab.com/
+:url-gitlab-int: https://docs.gitlab.com/ce/administration/integration/kroki.html
+:url-sphinx: https://www.sphinx-doc.org/
+:url-sphinx-int: https://github.com/sphinx-contrib/kroki
+:url-conf-kroki: https://marketplace.atlassian.com/apps/1224570
```
Comment (Mogztter) [inline docs/modules/setup/pages/third-party-tools.adoc:13]:
```suggestion
```

**Comment by cmorty** [comment]
@Mogztter : Sorry for not coming back to this earlier. I got a bit buried in work. Should I merge the first to commits (or all of them)?

**Comment by Mogztter** [comment]
@cmorty no need I can squash commits before merging, thanks for your contribution

**Pull request merged**